spec/controllers/users/phone_confirmation_controller_spec.rb

```diff
@@ -89,6 +92,18 @@
         it 'displays success flash notice' do
           expect(flash[:success]).to eq t('notices.phone_confirmation_successful')
         end
+
+        it 'tracks the update and confirmation event' do
+          expect(@analytics).to have_received(:track_event).
+            with('User confirmed their phone number')
+          expect(@analytics).to have_received(:track_event).
+            with('User changed and confirmed their phone number')
+        end
+
+        it 'creates a new :phone_changed event' do
+          expect(subject).to have_received(:create_user_event).with(:phone_changed)
+          expect(subject).to have_received(:create_user_event).exactly(:once)
+        end
```

Comment:
Add these expectations to an existing `it` block, to avoid slowing down this test even more? Or at least combine these 2 new ones? If a controller action has this many expectations, it's probably a sign that it's doing too much and that perhaps some new classes should be created to handle stuff that's not specific to a controller. `assign_phone` is a great candidate for that, but that's for another PR.